pkg/webreg: add a page with useful links

bbguimaraes opened the pull request:
These are the ones that were requested last week and some others I can think of
off the top of my head.

https://server-ci-tools-1206.apps.ci.l2s4.p1.openshiftapps.com

petr-muller commented on pkg/webreg/webreg.go:
```diff
@@ -1966,6 +1967,94 @@ const staticBoskosConfig = `resources:
+  </li>
+  <li>
+    <a href="https://console-openshift-console.apps.build01.ci.devcluster.openshift.com/"><code>build01</code></a>:
+    Openshift 4.x cluster in AWS that executes a growing subset of the jobs.
```
Maybe mention build02 and vsphere clusters too?

petr-muller commented on pkg/webreg/webreg.go:
```diff
@@ -1966,6 +1967,94 @@ const staticBoskosConfig = `resources:
+      </li>
+      <li>
+        <code>#forum-testplatform</code>: general queries and discussion for
+        the test platform.
```
...test platform. For general assistance, ping @dptp-helpdesk. For reporting an outage, ping @dptp-triage.

bbguimaraes commented:
- added `build02` and `vsphere` clusters (I omitted the console link for the latter because AFAIK it is not open to all users)
- changed console URLs for `build*` clusters to `build*.ci.openshift.org`
- added aliases to `#forum-testplatform` section
- added `#forum-release-controller` to list of Slack channels
- updated the test deployment
